debugging_output.md

```diff
@@ -0,0 +1,77 @@
+The TFLM debugging output tools allow TFLM users to easily debug their models
+by providing a tool that will compare the intermediate  values(output of each OP/Kernel)
+from a model post invoke between the TFLM and TfLite. As well as a way to
+compare intermediate values between TFLM x86 implementations and Optimized 
+Implementations.
+
+
+The workflow can be divided into two parts:
```

Comment:
This section describes the two stages of the embedded target debugging, but I think we should call out the two workflows in two sections:

```
## How to debug TFLM Python Interpreter output

This section would describe how to debug incorrect output from the python 
interpreter. This is relevant to cases where there is an existing model that is 
known to work with TFLite, but the output doesn't match for TFLM. 

## How to debug TFLM Interpreter output on embedded targets

This section would break down the two stage workflow for embedded targets
using a simulator.

Reply:
Done !